samples/WindowsAlarmsAndClock/Pages/EditAlarmPage.cs

```diff
@@ -0,0 +1,114 @@
+namespace WindowsAlarmsAndClock.Pages
+{
+    using System;
+
+    using Legerity.Pages;
+    using Legerity.Windows.Elements.Core;
+    using Legerity.Windows.Extensions;
+
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Appium.Windows;
+
+    /// <summary>
+    /// Defines the add/edit alarm page of the Windows Alarms & Clock application.
```

Comment:
Same as previous comment, this should be `&amp;`

Reply:
Sorted :) (as above)